mathics/builtin/forms/default_boxing.py

```diff
@@ -0,0 +1,39 @@
+"""
+Registers boxing routines for a number of the standard Builtin forms and Builtin Symbols
+"""
+
+from mathics.builtin.box.layout import (
+    SymbolFractionBox,
+    SymbolSqrtBox,
+    SymbolSuperscriptBox,
+)
+from mathics.builtin.forms.output import StandardForm, TraditionalForm
+from mathics.core.expression import Expression
+from mathics.core.symbols import SymbolDivide
+from mathics.core.systemsymbols import SymbolPower, SymbolSqrt
+
+
+def box_divide(expr: Expression, evaluation):
+    assert (
+        len(expr.elements) == 2
+    ), "Fraction Box element {elements} should have two items"
+    return Expression(SymbolFractionBox, *expr.elements)
+
+
+def box_power(expr: Expression, evaluation):
+    assert (
+        len(expr.elements) == 2
+    ), "Fraction Box element {elements} should have two items"
+    return Expression(SymbolSuperscriptBox, *expr.elements)
+
+
+def box_sqrt(expr: Expression, evaluation):
+    assert len(expr.elements) == 1, "Sqrt Box element {elements} should have one item"
+    return Expression(SymbolSqrtBox, *expr.elements)
+
+
+# More general, should we iterate over $BoxForms?
```

Comment:
I think this is not necesary. To populate  `$BoxForms` and make it effective, you should set the `ParentForm` to the new "BoxForm"  :
https://reference.wolfram.com/language/ref/message/ParentForm/deflt.html

So, if MakeBoxes receives a form not in `StandardForm`, `TraditionalForm`  as a second argument, then it should look at the parent form of this new form.